scripts/pre-commit-hooks/stack-yml-generated-is-valid/launch.bash

```diff
@@ -51,17 +51,11 @@ rm "$repo_basedir"/docker-compose.yml 2>/dev/null || true
 rm "$repo_basedir"/.env-devel 2>/dev/null || true
 # via https://stackoverflow.com/a/2466755
 rm -rf osparc-simcore 2>/dev/null || true
-git clone "${GIT_SIMCORE_REPO_URL}"
+git clone https://github.com/ITISFoundation/osparc-simcore
```

Comment:
TIP:  When you are interested only in the latest version & branch of a repo, do instead a shallow clone ``git clone --depth=1``. It would be faster and needs to download much less

SEE https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt

Reply:
Done. Thank you for the remark!